src/schema/v2/artist/insights.ts

```diff
@@ -36,6 +37,17 @@ export const ArtistInsight = new GraphQLObjectType<any, ResolverContext>({
     },
     description: {
       type: GraphQLString,
+      args: {
+        ...markdown().args,
+        defaultValue: {
+          type: GraphQLString,
+          defaultValue: "plain",
```

Comment:
So this isn't specified correctly. You want to merge into the markdown args and provide a `defaultValue` to the `format` arg. It shoudn't be a string but will rather be of type `Format` with a value of `PLAIN`

Reply:
I'm not quite sure if that's possible. Do you have an example?

Reply:
Something like this would work?

```
--- a/src/schema/v2/artist/insights.ts
+++ b/src/schema/v2/artist/insights.ts
@@ -14,7 +14,8 @@ import {
   getArtistInsights,
   getAuctionRecord,
 } from "./helpers"
-import { markdown, formatMarkdownValue } from "schema/v2/fields/markdown"
+import { formatMarkdownValue } from "schema/v2/fields/markdown"
+import Format, { FORMATS } from "schema/v2/input_fields/format"
 
 export const ArtistInsightKind = new GraphQLEnumType({
   name: "ArtistInsightKind",
@@ -38,14 +39,12 @@ export const ArtistInsight = new GraphQLObjectType<any, ResolverContext>({
     description: {
       type: GraphQLString,
       args: {
-        ...markdown().args,
-        defaultValue: {
-          type: GraphQLString,
-          defaultValue: "plain",
+        format: {
+          ...Format,
+          defaultValue: FORMATS.PLAIN.value,
         },
       },
       resolve: ({ description }, { format }) => {
-        if (!format) return formatMarkdownValue(description, "plain").trim()
         return formatMarkdownValue(description, format).trim()
       },
     },
diff --git a/src/schema/v2/input_fields/format.ts b/src/schema/v2/input_fields/format.ts
index e9df54f1..562234d1 100644
--- a/src/schema/v2/input_fields/format.ts
+++ b/src/schema/v2/input_fields/format.ts
@@ -1,12 +1,12 @@
-import { GraphQLEnumType } from "graphql"
+import { GraphQLEnumType, GraphQLArgumentConfig } from "graphql"
 
-const FORMATS = {
+export const FORMATS = {
   HTML: { value: "html" },
   PLAIN: { value: "plain" },
   MARKDOWN: { value: "markdown" },
 } as const
 
-const Format = {
+export const Format: GraphQLArgumentConfig = {
   type: new GraphQLEnumType({
     name: "Format",
     values: FORMATS,


```